ocs_ci/utility/version.py

```
@@ -150,25 +151,62 @@ def get_ocp_version(seperator=None):
     payload = {"channel": f"stable-{channel}"}
-    r = requests.get(url, headers=headers, params=payload, timeout=120)
-    nodes = r.json()["nodes"]
+
+    try:
+        r = requests.get(url, headers=headers, params=payload, timeout=120)
+        r.raise_for_status()
+        nodes = r.json()["nodes"]
+    except requests.exceptions.HTTPError as http_err:
+        log.error(
+            f"HTTP error {r.status_code} when retrieving GA version for channel {channel}: {http_err}"
+        )
```

Ruff (0.15.20)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)